src/mmo/socket.js

```diff
@@ -94,34 +94,6 @@ const measureLatency = (ping_proto) => {
 
 socket.onopen = () => {
 
-    if (url.searchParams.has('code')) {
-
-        /// send access code to server
-        const state = JSON.parse(decodeURIComponent(url.searchParams.get("state")))
-        const accessCodeMsg = new AccessCodeMsg()
-        accessCodeMsg.setAccessCode(url.searchParams.get('code'))
-        accessCodeMsg.setType(state.type)
-        const initializationMsg = new InitializationMsg()
-        initializationMsg.setAccessCodeMsg(accessCodeMsg)
-        const sm64jsMsg = new Sm64JsMsg()
-        sm64jsMsg.setInitializationMsg(initializationMsg)
-        const rootMsg = new RootMsg()
-        rootMsg.setUncompressedSm64jsMsg(sm64jsMsg)
-        sendData(rootMsg.serializeBinary())
-    } else if (process.env.PRODUCTION == 0) {
```

Comment:
should I be adding an easier way for local development? I deleted this, but haven't added it to the REST method

Reply:
yea, we'll want to make sure there is a way to test MMO locally without having to setup a bunch of Oauth or other prod stuff